### ISSUE TITLE
Add ffmpeg and ffprobe to docker

### ISSUE THREAD

**Issue by gcraenen** [nzbgetcom/nzbget#409]
### Is there already an issue for this request?

- [X] I have checked older issues, open and closed

### Platform

Linux/Docker

### Describe the enhancement you'd like

Please add ffmpeg and ffprobe packages to the docker image for use with the Failure link script.

**Comment by luckedea** [comment]
Yes, good idea, but we'll take it further - if you are installing FailureLink using Extension Manager (EM) - it would suggest to download missing dependencies automatically.

**Comment by muhlman** [comment]
I think it would be a bigger undertaking than even this, but I would love to get gpu passthrough for nvidia working as well so I can run post processing scripts that transcode the video.
